src/server.h

```diff
@@ -1010,15 +1010,22 @@ typedef struct blockingState {
                            * is > timeout then the operation timed out. */
     int unblock_on_nokey; /* Whether to unblock the client when at least one of the keys
                              is deleted or does not exist anymore */
+    union {
+        listNode *client_waiting_acks_list_node; /* list node in server.clients_waiting_acks list. */
+        listNode *postponed_list_node;           /* list node within the postponed list */
```

Comment:
It's more clear to describe exactly which list this is, just like the line above. (I'm not very familiar with it.)

```suggestion
        listNode *postponed_list_node;           /* list node in server.postponed_clients */
```